src/common/mpi.jl

```diff
@@ -0,0 +1,49 @@
+# Convenience functions for working with MPI
+using MPI
```

Comment:
If you prepend everything by `mpi_` I wonder if it would not be better to put this into a separate module?

Reply:
We did that for smearing and I'm not sure I like how that turned out. We definitely don't want to confuse mpi_sum with sum (they are pretty different)

Reply:
That's true. Also agree with the smearing aspect actually ...